plugins/plugin-k8s/src/test/k8s1/helm.ts

```diff
@@ -28,4 +30,25 @@ describe('helm commands', function (this: common.ISuite) {
       .then(cli.expectBlank)
       .catch(common.oops(this)))
   })
+
+  it(`should create sample helm chart`, () =>
+    cli.do(`helm install --name test-release stable/mysql`, this.app)
+      .then(async (res) => {
+        await cli.expectOKWithAny(res)
+
+        const text = await this.app.client.getText(` .streaming-output`)
+
+        const table = await this.app.client.getText(`.result-table-title`)
+        let ok = true
+        for (const t of text) {
+          if (t !== '') {
+            if (!t.includes('NOTES:') && !t.includes('LAST DEPLOYED:')) { ok = false }
+          }
+        }
+        return assert.ok(table.length === 6 && ok)
+      })
+      .then(() => cli.do(`helm delete --purge test-release`, this.app))
```

Comment:
Can you separate the deletion into a new test case? By doing so, we can have a better notion of which command fails. In your change, even if the deletion fails, what developers would see is `should create sample helm chart` fails. Thanks!
```
it(`should create sample helm chart`, () => cli.do(`helm install --name test-release stable/mysql`, this.app)
.then(...)
.catch(...))

it('should delete sample helm chart', () => cli.do(`helm delete --purge test-release`, this.app)
.then(...)
.catch(...))
```

Reply:
Also,  after helm delete test, can you test the helm list turns to empty again, (just like what we do in the beginning of the test suite)?  
```
it(`should list empty releases via helm list`, () => cli.do(`helm list`, this.app)
      .then(cli.expectBlank)
      .catch(common.oops(this)))
```